pkg/github/client/client.go

```diff
@@ -23,6 +23,8 @@ type Client interface {
 	ListIssueComments(issue scm.RepositoryIssue) ([]*gogh.IssueComment, error)
 	CreateIssueComment(issue scm.RepositoryIssue, commentMsg *string) error
 	CreateStatus(change scm.RepositoryChange, repoStatus *gogh.RepoStatus) error
+	AddLabelToPullRequest(owner, repo string, prNumber int, label []string) ([]*gogh.Label, error)
+	RemoveLabelFromPullRequest(owner, repo string, prNumber int, label string) error
```

Comment:
These methods should have been implemented also in other implementations of the Client interface